labs-ios-starter/Model/Topic.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  Topic.swift
+//  labs-ios-starter
+//
+//  Created by Tobi Kuyoro on 10/09/2020.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import Foundation
+
+class Topic: Codable {
+    let topicId: Int?
+    let userId: Int?
+    let title: String
+    let surveys: [Int]?
+    let users: [User]?
+    var dateCreated: Date? = nil
+    var lastModified: Date? = nil
+    var frequency: String? = nil
+    var contextType: Context? = nil
+    var leaderQuestions: [Question]? = nil
+    var memberQuestions: [Question]? = nil
```

Comment:
Why are these initialised with nil values? Are you sure you need them in your model?